spec/support/idv_examples/usps_verification.rb

```diff
@@ -0,0 +1,46 @@
+shared_examples 'signing in with pending USPS verification' do |sp|
+  it 'prompts for confirmation code at sign in' do
+    otp = 'abc123'
+    profile = create(
+      :profile,
+      deactivation_reason: :verification_pending,
+      phone_confirmed: false,
+      pii: { otp: otp, ssn: '123-45-6789', dob: '1970-01-01' }
+    )
+    user = profile.user
+
+    allow(FeatureManagement).to receive(:reveal_usps_code?).and_return(true)
+
+    if sp == :saml
```

Comment:
This is pretty similar to the code with have in the account creation shared example. It'd be nice to move all of this into an idv helper